test/unit/DCAFeeManager/dca-fee-manager.spec.ts

```diff
@@ -6,7 +6,7 @@ import { DCAFeeManagerMock, DCAFeeManagerMock__factory, IDCAHub, IDCAHubPosition
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 import { duration } from 'moment';
 import { behaviours, wallet } from '@test-utils';
-import { IDCAFeeManager } from '@typechained/DCAFeeManager';
+import { IDCAFeeManager } from '@typechained/contracts/DCAFeeManager/DCAFeeManager';
```

Comment:
I don't understand how typechain decides the folder structure... 🤦🏽‍♂️

Reply:
Me neither 🤷